tests/data/sentinel2/data.py

```diff
@@ -11,140 +11,45 @@
 from rasterio.crs import CRS
 
 SIZE = 128
+current_x_offset = 0
 
 np.random.seed(0)
 
-FILENAME_HIERARCHY = dict[str, "FILENAME_HIERARCHY"] | list[str]
-
-filenames: FILENAME_HIERARCHY = {
-    # USGS Earth Explorer
```

Comment:
I don't like that we're getting rid of these. These were added because EarthExplorer and Copernicus use very different file structures to organize the data, and we want to test to ensure that both remain supported.